__tests__/app.test.js

```diff
@@ -0,0 +1,53 @@
+const app = require("../app");
+const request = require("supertest");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+    return connection.end();
+});
+
+beforeEach(() => {
+    return seed(data);
+});
+
+describe('app', () => {
+    describe('/api/healthcheck', () => {
+        test('200 : responds with a status of 200', () => {
+            return request(app)
+            .get('/api/healthcheck')
+            .expect(200)
+            .then((response) => {
+                const { msg } = response.body;
+                expect(msg).toBe('Server is running')
+            });
+        });
+    });
+    describe('/api/topics', () => {
+        test('200 : responds with an array of topic objects', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const { topics } = response.body;
+                expect(topics).toBeInstanceOf(Array);
+                expect(topics).toHaveLength(3);
+                topics.forEach((topic) => {
+                    expect(topic).toHaveProperty('slug', expect.any(String));
+                    expect(topic).toHaveProperty('description', expect.any(String));
+                });
+            });
+        });
+        test('404 : returns an error message if path is incorrect', () => {
```

Comment:
Should this test be part of the /api/topics describeblock?

The enpoint you are sedning your request to is topics123 but could be anything and doesn't only relate to a request for topics.

I think this test is good to do now but should be outside of the /api/topics describe block